### PR TITLE
Integrated E2E tests into CI workflow.

### PR DESCRIPTION
**Pull request by U-DON**
## Description
Moved Cypress and Nighwatch E2E tests into the main GitHub Actions CI workflow.

## Testing done
Verified passing CI runs.

## Acceptance criteria
- [ ] Cypress and Nightwatch E2E tests should be consolidated into the main CI workflow.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -195,112 +340,88 @@ jobs:
...
-        run: yarn build --buildtype=${{ matrix.buildtype }} --drupal-address="https://cms-vets-website-branch-builds-lo9uhqj18nwixunsjadvjsynuni7kk1u.ci.cms.va.gov" --no-drupal-proxy
-        timeout-minutes: 30
-        env:
-          NODE_ENV: production
-
-      - name: Check broken links
-        id: get-broken-link-info
-        run: node ./script/github-actions/check-broken-links.js ${{ matrix.buildtype }}
+      - name: Start server
+        run: node src/platform/testing/e2e/test-server.js --buildtype vagovprod --port=3002 &
```
Comment (U-DON) [inline .github/workflows/continuous-integration.yml:344]: Since this is in a self-hosted runner, probably need port randomization here too, but will leave that for a separate PR if necessary.

Comment (ddzz) [reply .github/workflows/continuous-integration.yml:344]: Yeah, we'll need port randomization. I can open a PR for that.

**Review by ddzz — APPROVED** [review]
LGTM

**Review by k80bowman — APPROVED** [review]
